qadence/backends/pulser/pulses.py

```diff
@@ -42,6 +42,54 @@
 ]
 
 
+def add_addressing_pattern(
+    sequence: Sequence,
+    config: Configuration,
+) -> None:
+    total_duration = sequence.get_duration()
+    n_qubits = len(sequence.register.qubits)
+
+    support = tuple(range(n_qubits))
+    if config.addressing_pattern is not None:
+        amp = config.addressing_pattern.amp
+        det = config.addressing_pattern.det
+        weights_amp = config.addressing_pattern.weights_amp
+        weights_det = config.addressing_pattern.weights_det
```

Comment:
Is this assuming that the pattern weights are given as numbers for the Pulser backend?

Reply:
Currently, weights for the Pulser backend can only be fixed. I'll open a separate issue for that since it will require more changes.

Reply:
Great, makes sense. Could you please add a comment on that? It's quite important. Then go ahead with merging this one.